api/queries_issue.go

```diff
@@ -36,6 +36,7 @@ type Issue struct {
 	Assignees      Assignees
 	Labels         Labels
 	ProjectCards   ProjectCards
+	ProjectItems   ProjectItems
```

Comment:
:thought_balloon: Pre-fetch references to items to avoid expensive lookup later on. We need the link because the entity ID does not match the ID of the corresponding item in Project V2. And project's item's ID is required to unlink item from the project.